src/components/Button/index.js

```diff
@@ -0,0 +1,30 @@
+import './styles.css';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const colors = {
+  gray: 'rgb(224, 224, 224)',
+  orange: 'rgb(236, 146, 67)',
+}
+
+const Button = (props) => {
```

Comment:
We can perform the deconstruction right in the argument area:

```javascript
const Button = ({ color, wide, name, handleClick }) => (
  <button
  // ditto
 </button>
);
```

Reply:
It's refactored.